backend/models.py

```diff
@@ -2,3 +2,6 @@
 from django.contrib.auth.models import AbstractUser
 
 # Create your models here.
+class Task(models.Model):
```

Comment:
I've noticed that you don't have the ID field here which is ok, as Django has a default ID field for each table.